subprojects/docs/src/docs/userguide/troubleshooting/validation_problems.adoc

```diff
@@ -213,16 +213,6 @@ If you use the `@Input` annotation on a file-based property, Gradle wouldn't con
 To fix this problem, you need to tell Gradle if the file property represents an input file, in which case you should annotate it with `@InputFile`, or a directory, in which case it should be annotated with `@InputDirectory`.
 If what you really wanted to say is that the actual file path is an input, then you should return a `String` instead which corresponds to the absolute path of the file.
 
-[[unresolvable_input]]
```

Comment:
What is the exception normally thrown now? Would it help users if we modify this with description of an error here instead of removing it?

Reply:
I was wondering that. Though in general you shouldn't add anything as an input which can't be resolved. Gradle already fails when your task has an output in that case even without those changes. So we only cover more cases. I don't think it is worth to keep this section, since the failure will directly point to the input property which can't be resolved.